processor/metricstransformprocessor/metrics_transform_processor_otlp.go

```diff
@@ -541,7 +554,13 @@ func transformMetric(metric pmetric.Metric, transform internalTransform) bool {
 			updateLabelOp(metric, op, transform.MetricIncludeFilter)
 		case aggregateLabels:
 			if canChangeMetric {
-				aggregateLabelsOp(metric, op)
+				ops := []string{}
+				for k, v := range op.labelSetMap {
+					if v {
+						ops = append(ops, k)
+					}
+				}
+				aggregateLabelsOp(metric, ops, op.configOperation.AggregationType)
```

Comment:
```suggestion
				attrs := []string{}
				for k, v := range op.labelSetMap {
					if v {
						attrs = append(attrs, k)
					}
				}
				aggregateLabelsOp(metric, attrs, op.configOperation.AggregationType)
```
This is a list of attributes and not operations, right?